doc/examples/filters/plot_enhancement_measure.py

```diff
@@ -0,0 +1,64 @@
+"""
+=========================================
+Quantitative Measure of Image Enhancement
+=========================================
+
+In this example, we will measure the result of applying contrast enhancement
+to the image. We will use a measure called EME [1]_.
+
+It is defined as a log of the ratio of local maximum to local minimum
+within a sliding window. The code allows the user to choose the window size.
+This function was defined to quantify improvement of the image after processing.
+
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from skimage import data
+from skimage.filters import gaussian
+from skimage.exposure import equalize_adapthist
+from skimage.measure.simple_metrics import enhancement_measure
```

Comment:
We don't typically do nested level imports in scikit-image.  Please import directly from `skimage.measure`.